src/main/kotlin/tech/relaycorp/relaynet/keystores/CertificateStore.kt

```diff
@@ -8,43 +8,55 @@ import tech.relaycorp.relaynet.wrappers.x509.Certificate
 
 abstract class CertificateStore {
 
+    enum class Scope(val value: String) {
```

Comment:
For consistency with the rest of the library, can you please make this a top level definition named `CertificateScope`?